tests/regression.rs

```diff
@@ -622,3 +622,107 @@ fn test_hough_line_detection() {
         assert_pixels_eq!(lines_image, truth);
     }
 }
+
```

Comment:
The tests in this file all compare generated images to images under tests/data/truth.

Tests that don't do I/O should go in the same file as the code they're testing.